onboarding/source/amb_light_service.c

```diff
@@ -6,7 +6,15 @@
 
 /* USER CODE BEGIN */
 // Include any additional headers and global variables here
+#include "FreeRTOS.h"
+#include "os_task.h"
+#include "os_queue.h"
 
+void adcStartConversion_selChn(adcBASE_t *adc, unsigned channel, unsigned fifo_size, unsigned group);
```

Comment:
It's clearer if you write the types as `unsigned int` instead of `unsigned`. It's even better if you use the appropriate fixed-size integer types for these arguments (such as `uint8_t`).

Reply:
Done